documentation/bolt_configuration_options.md

```diff
@@ -50,6 +50,7 @@ ssh:
 -   `proxyjump`: A jump host to proxy SSH connections through, and an optional user to connect with, for example: jump.example.com or user1@jump.example.com.
 -   `run-as`: A different user to run commands as after login.
 -   `run-as-command`: The command to elevate permissions. Bolt appends the user and command strings to the configured run as a command before running it on the target. This command must not require an interactive password prompt, and the `sudo-password` option is ignored when `run-as-command` is specified. The run-as command must be specified as an array.
+-   `sudo-executable`: The executable to use to run as another user when using `run-as`. This is used to replace the executable used when escalating with a password is desired, as the `run-as-command` option does not support escalating with a password. The command run on the target is `$sudo-executable -S -u $user -p $sudo-password`. **Note:** This option is experimental.
```

Comment:
```suggestion
-   `sudo-executable`: The executable to use when escalating to the configured `run-as` user. This is useful when you want to escalate using the configured `sudo-password`, since `run-as-command` does not use `sudo-password` or support prompting. The command executed on the target is `<sudo-executable> -S -u <user> -p custom_bolt_prompt <command>`. **Note:** This option is experimental.
```